CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+# Code de Conduite Contributeurs
+
+En tant que contributeurs et responsables de ce projet, et dans l'intérêt
+de favoriser une communauté ouverte et accueillante, nous nous engageons à
+respecter toutes les personnes qui contribuent en rapportant des erreurs,
+en postant des demandes de fonctionnalités nouvelles, en mettant à jour la
+documentation, en soumettant des pull requests ou des correctifs, ainsi que
+toutes autres activités.
+
+Nous sommes déterminés à rendre toute participation à ce projet une
+expérience exempte de harcèlement pour tout le monde, quel que soit le niveau
+d'expérience, le sexe, l'identité ou l'expression de genre, l'orientation
+sexuelle, le handicap, l'apparence personnelle, la taille physique, la race,
+l'origine ethnique, l'âge, la religion ou la nationalité.
+
+De tels comportements ne sont pas acceptables, par exemple :
+
+* l'utilisation de langage ou d'imagerie sexualisé ;
+* les attaques personnelles ;
+* les trolls ou commentaires insultants ou désobligeants ;
+* le harcèlement en public ou en privé ;
+* la publication d'autres informations privées, telles que les adresses
+  physiques ou électroniques, sans permission explicite ;
+* toute autre conduite contraire à l'éthique ou non professionnelle.
+
+Les mainteneurs du projet ont le droit et la responsabilité de supprimer,
+modifier ou rejeter les commentaires, commits, code, modifications du wiki,
+questions et autres contributions qui ne respectent pas ce Code de Conduite,
+ou de bannir temporairement ou définitivement tout contributeur à la suite
+d'autres comportements qu'ils jugent inappropriés, menaçants, injurieux,
+ou nuisibles.
+
+En adoptant ce Code de Conduite, les mainteneurs du projet s'engagent à
+appliquer équitablement et uniformément ces principes à tous les aspects de
+la gestion de ce projet. Les mainteneurs de projets qui ne suivent pas ou ne
+font pas respecter le Code de Conduite peuvent être retirés de façon permanente
+de l'équipe de projet.
+
+Ce Code de Conduite s'applique à la fois au sein des espaces de projet
+ainsi que dans les espaces publics quand un individu représente le projet
+ou sa communauté.
+
+Les instances de comportement abusif, harcelant ou autrement inacceptable
+peuvent être signalés en contactant un responsable de projet à
+zestedesavoir at gmail.com. Toutes les plaintes seront examinées et étudiées
+et se traduiront par une réponse jugée nécessaire et appropriée aux
+circonstances. Les mainteneurs s'obligent à garder confidentielles les
+informations de la personne qui remonte un incident.
+
+Ce Code de Conduite est adaptée du [Contributor Covenant](http://contributor-covenant.org),
+version 1.3.0, [disponible ici](http://contributor-covenant.org/version/1/3/0/fr).
```

Comment:
Depuis leurs page, on peut lire «Important! You must add a contact method to the placeholder in the document so that people know how to report violations.». Il manque ça.